Src/Base/AMReX_FArrayBox.cpp

```diff
@@ -769,7 +769,7 @@ FABio_8bit::write (std::ostream&    os,
 {
     BL_ASSERT(comp >= 0 && num_comp >= 1 && (comp+num_comp) <= f.nComp());
 
-    const Real eps = Real(1.0e-8); // FIXME - whats a better value?
+    const Real eps = 1.0e-8_rt; // FIXME - whats a better value?
```

Comment:
Btw, a better value might be [`std::numeric_limits<Real>::epsilon`](https://en.cppreference.com/w/cpp/types/numeric_limits/epsilon), without checking what this function actually does.